lib/bundle.js

```diff
@@ -186,13 +189,19 @@ exports.bundle = function(moduleExpression, fileName, opts) {
         return path.resolve(config.pjson.baseURL, file);
       }), function invalidate(invalidated) {
         systemBuilder.invalidate(toFileURL(invalidated));
-      }, function rebuild() {
-        return bundle();
+      }, function rebuild(triggerFile) {
+        if (triggerFile != fileName &&
+            triggerFile.indexOf('jspm.browser.js') == -1) {
```

Comment:
This is just an example to show the possibilities here. Do you prefer rebuilding on any file change or just the change of file in the bundle? As I can see jspm.browser.js isn't a part of the bundle, so it won't trigger rebuild. We can add it later after we address the bug that leads to this file being touched on rebuild.